pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -528,6 +530,9 @@ func (v *vpnShoot) podTemplate(serviceAccount *corev1.ServiceAccount, secrets []
 				v1beta1constants.LabelApp:       LabelValue,
 				managedresources.LabelKeyOrigin: managedresources.LabelValueGardener,
 				"type":                          "tunnel",
+				// ManagedSeeds run both, the VPN shoot client and Istiod in the same cluster. Thus, we need the following
+				// label to allow the VPN shoot client to reach Istiod's validating webhook.
+				gardenerutils.NetworkPolicyLabel(istio.IstiodServiceName, istio.PortWebhookServer): v1beta1constants.LabelNetworkPolicyAllowed,
```

Comment:
not needed since we already have https://github.com/gardener/gardener/pull/7570/files#diff-d209ad66b99a63c3b888ee941c5fc40f2fd54823e01bc353045551af0a6850a2R7

Reply:
Tackled in 37f5002d42f1b0f204cf9f074922f5bd493a7bba.